Sources/TuistSigning/Linter/SigningLinter.swift

```diff
@@ -40,8 +40,13 @@ final class SigningLinter: SigningLinting {
             """,
             severity: .error
         )
+        let buildSettingRegex = "\\$\\((.*)\\)"
```

Comment:
Note that there are two options for variables: one is `$(VARIABLE)` and the other is `${VARIABLE}`. This pattern will only catch `$(VARIABLE)` option. You'll need to modify the regex accordingly (changing `(` to `[\({]` and similarly for the closing bracket)